torch/csrc/jit/pybind_utils.h

```diff
@@ -446,6 +446,11 @@ inline IValue toIValue(
       return userObj;
     }
     case TypeKind::NumberType:
+      if (py::isinstance<py::int_>(obj)) {
```

Comment:
What's the context on the schema you're using that needs to use a `Scalar` type?

The changes seem fine otherwise

Reply:
I first used `torch::jit::getAllOperatorsFor` to get an operator list for `aten::add`, then trying to match one by passing  (`torch.ones(2, 2)`, `2`) as the `py::args` to `torch::jit::createStackForSchema`.

Reply:
The arg `2` is somehow recognized as a `NumberType` instead of an `IntType`.